unittest/python/python-pizza.py

```diff
@@ -0,0 +1,149 @@
+import os
+import sys
+import unittest
+from lammps import lammps
+
+EXAMPLES_DIR=os.path.abspath(os.path.join(__file__, '..', '..', '..', 'examples'))
+PIZZA_DIR=os.path.abspath(os.path.join(__file__, '..', '..', '..', 'tools', 'python', 'pizza'))
```

Comment:
@akohlmey What do you think about making `pizza` a real python package with its own `setup.py`? And then change its usage to:
```python
from pizza import dump, log
```

Reply:
I don't really want to sink more time into this. It already is more than I wanted.
I would suggest to do this in the https://github.com/lammps/pizza repository. The Python 3 changes here need to be carried over there anyway. But you'll have to discuss this with @sjplimp. So far the design goal has been to make this more a collection of bits an pieces that can be used as needed, but without being a regular package.

The other idea that had been kicked around would be to move the pizza folder(s) and content to `python/lammps/pizza` and make it part of the LAMMPS module.

As mentioned above, I think both exceeds the scope of this PR and is a job for somebody else with more interest in Python programming and LAMMPS pre/post processing.